### DISCUSSION TITLE
漫游的服务器没办法正常搜索

### DISCUSSION THREAD

**Discussion by kettly1260** [ywmoyue/biliuwp-lite#30] [Q&A]
大家有使用漫游代理吗？我发现我用漫游的服务器没办法正常搜索港澳台番剧和视频。更换了全部代理都不行

**Comment by ywmoyue** [comment]
是bug

**Reply by Playfulnurse** [reply]
非常感谢老哥，这个有时候滚轮向下刷新视频会闪退，之前有次都没打开就闪退

**Comment by ywmoyue** [comment]
我这边又没复现出来了，你还有这情况吗，有的话有提示什么错误吗

**Reply by kettly1260** [reply]
首先是搜索结果很少，很多搜索不到，其次是搜索到的片源点进去之后无法播放，提示
`
读取视频播放地址失败
请求信息：
WebAPI-无代理：抱歉您所在地区不可观看！
WebAPI-香港代理：服务器内部错误:-663错误,被鼠鼠制裁了,请稍后重试
WebAPI-台湾代理：服务器内部错误:-663错误,被鼠鼠制裁了,请稍后重试
WebAPI-大陆代理：服务器内部错误:-663错误,被鼠鼠制裁了,请稍后重试
[/vl.PlayURL/PlayView]:请求失败,没有数据返回
`
但是手机用这些公共代理是正常的

**Reply by ywmoyue** [reply]
问下你用的是最新版吗
因为播放漫游视频报663错误是我4.4.10就修复了的问题，后续我本地没再出现过 https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.4.10

如果已经是最新版，建议尝试重新登录一次

**Reply by kettly1260** [reply]
用的是4.5.3，登录过几次了，不懂到底是什么问题

**Reply by ywmoyue** [reply]
我这边优化下日志处理，之后我再看日志分析吧

**Comment by modebra** [comment]
我也是正常设置了代理，在网页是正常可用的，版本是4.5.3。点开漫游视频的时候提示：
读取视频播放地址失败
请求信息:
WebAPI-无代理:抱歉您所在地区不可观看!
WebAPI-香港代理: Object reference not set to an instance of an object.
WebAPI-台湾代理: Object reference not set to an instance of an object.
WebAPI-大陆代理:抱歉您所在地区不可观看!
[/v1.PlayURL/PlayView]:请求失败没有数据返回

**Reply by modebra** [reply]
按照这个网址https://www.jianshu.com/p/9d1566aa94cf的教程解除了 UWP 应用的网络隔离，没有效果

**Comment by ywmoyue** [comment]
请在更新4.5.4后参考这篇内容将复现问题时候的日志文件发送到我邮箱 https://github.com/ywmoyue/biliuwp-lite/discussions/45
@modebra @kettly1260

**Comment by ywmoyue** [comment]
麻烦测试一下漫游搜索和播放是否可用：https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.6.4.3
